web/src/components/Layout.tsx

```diff
@@ -110,14 +119,23 @@ type PropsType = {
   modal?: ReactNode
   children?: ReactNode
   fullWidth?: boolean
+  scrollable?: boolean
 }
 
 /**
  * The standard Layout, used for any view in this app as a container.
  * If a footer is supplied and there's not enough content (in header and children) to fill the viewbox, the footer will
  * always stick to the bottom of the viewbox.
  */
-const Layout = ({ footer, header, toolbar, modal, children, fullWidth = false }: PropsType): JSX.Element => {
+const Layout = ({
+  footer,
+  header,
+  toolbar,
+  modal,
+  children,
+  fullWidth = false,
+  scrollable = true
```

Comment:
Since setting this to false only for specific conditions this should be named differently, e.g. something like `disableScrollingSafari` or similar